package-lock.json

```diff
@@ -10,9 +10,9 @@
       "integrity": "sha512-6j62XnemaHbvJyGioOxt3pvmLbVWAe3askFjenWBLKTL/3Ni9unNbrQHFbwXtOfUpVahIdQmvHGszPMcqxQk7Q==",
       "dev": true,
       "requires": {
-        "broccoli-funnel": "^2.0.1",
-        "ember-cli-babel": "^6.12.0",
-        "ember-cli-htmlbars-inline-precompile": "^1.0.0"
+        "broccoli-funnel": "2.0.1",
```

Comment:
please use npm@6
and then 
npm cache clean --force
rm -rf node_modules
rm -rf package-lock.json
npm i